addons/schedule/model-java/src/main/java/org/commonjava/indy/schedule/datastax/ScheduleDBUtil.java

```diff
@@ -0,0 +1,33 @@
+package org.commonjava.indy.schedule.datastax;
+
+public class ScheduleDBUtil
+{
+
+    public static final String TABLE_SCHEDULE = "schedule";
+
+    public static String getSchemaCreateKeyspace( String keyspace )
+    {
+        return "CREATE KEYSPACE IF NOT EXISTS " + keyspace
+                        + " WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor':1};";
```

Comment:
> I had a question in there about the replication factor configuration.
> 
> Beyond that, I think we don't want to tie our scheduler support to the filesystem content. We disable repositories when we have network / upstream server problems, and want to schedule them for retry. We may also use this scheduler for other things in the future, such as cleaning up orphaned tracking data from failed builds.
> 
> I think it would be good to view the scheduler as a potential target for breaking into its own microservice, where we would publish events when things expire, and provide a scheduling API for other microservices to use. To do this, I think we'd need to look at the scheduling piece as separate from the content storage.

See, I will adjust the partition key that is currently tied to storeKey, the storekey should be part of the data rather than in a dedicate column I think.

Reply:
Or we can still use storekey since it's hard to create index on the dataMap if there are different objects in future. We can create different scheduler(schema) for the specific purpose.